BigQuery byte_length expression throws exception

**Describe the bug**
Attenpting a count or sum validation on a binary column will use the byte length as part of the expression.

On BigQuery this is failing:
```
E               google.api_core.exceptions.BadRequest: 400 Syntax error: Expected ")" but got ":" at [6:23]; reason: invalidQuery, location: query, message: Syntax error: Expected ")" but got ":" at [6:23]
E
E               Location: europe-west1
E               Job ID: a34ce483-8a9c-47a3-a5cb-ff174b671ebd
```

**What version of DVT are you using?**
8.1.0

**What type of connections are you using for source and target?**
BigQuery

--- a/tests/system/data_sources/test_oracle.py
+++ b/tests/system/data_sources/test_oracle.py
@@ -402,6 +402,21 @@ def test_column_validation_oracle_to_postgres():
     "data_validation.state_manager.StateManager.get_connection_config",
     new=mock_get_connection_config,
 )
+def test_column_validation_binary_to_bigquery():
+    """Oracle to BigQuery dvt_binary column validation."""
+    column_validation_test(
+        tables="pso_data_validator.dvt_binary",
+        tc="bq-conn",
+        count_cols="binary_id",
+        min_cols="binary_id",
+        sum_cols="binary_id",
+    )
+
+
+@mock.patch(
+    "data_validation.state_manager.StateManager.get_connection_config",
+    new=mock_get_connection_config,
+)
 def test_column_validation_large_decimals_to_bigquery():
     """Oracle to BigQuery dvt_large_decimals column validation."""
     cols = "col_dec_18,col_dec_38,col_dec_38_9,col_dec_38_30"

